### ISSUE TITLE
Seqfault when opening corrupted VCF

### ISSUE THREAD

**Issue by juliangehring** [brentp/cyvcf2#24]
Trying to open a corrupted VCF (or alternatively, a file that is no VCF at all) causes a segfault:
```python
cyvcf2.VCF("no-valid.vcf")
```

```
[E::bcf_hdr_read] invalid BCF2 magic string
[1]    89493 segmentation fault  ipython
```

I'm not sure if this is a rather a problem of htslib than cyvcf2, but is there a way to guard against this? Ideally, one could capture this as an IOError exception.

**Comment by brentp** [comment]
Thanks for reporting. I just pushed a fixed and tag v0.6.2 and also sent that off to pypi.

**Comment by juliangehring** [comment]
Thank you for the very fast fix.
